src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
```

Comment:
The package looks diff from the file path. So move the class to com/company package.